apps/i18n/common/en_us.json

```diff
@@ -195,6 +195,7 @@
   "shareWarningsAge": "Please provide your age below and click OK to continue.",
   "shareWarningsMoreInfo": "More Info",
   "shareWarningsStoreData": "This app built on Code Studio stores data that could be viewed by anyone with this sharing link, so be careful if you are asked to provide personal information.",
+  "showBlock": "Get a block",
```

Comment:
Note that we can restrict to en-us users by only analyzing that data or by only allowing them to participate in the experiment. The latter would give a better UX experience for non-en-us users (having [probabilistically more likely] translated strings), though I'm fine with either approach.